src/cyw43_ll.c

```diff
@@ -334,7 +334,7 @@ static uint32_t cyw43_read_backplane(cyw43_int_t *self, uint32_t addr, size_t si
     if (size == 4) {
         addr |= SBSDIO_SB_ACCESS_2_4B_FLAG;
     }
-    uint32_t reg = cyw43_read_reg(self, BACKPLANE_FUNCTION, addr, size);
+    uint32_t reg = cyw43_read_reg(self, BACKPLANE_FUNCTION, addr | SBSDIO_SB_ACCESS_2_4B_FLAG, size);
```

Comment:
This will need to be used only for SPI, and also combined with the above bitwise or.  Eg:
```c
#if CYW43_USE_SPI
addr |= SBSDIO_SB_ACCESS_2_4B_FLAG;
#else
if (size == 4) {
    addr |= SBSDIO_SB_ACCESS_2_4B_FLAG;
}
#endif
```